gninasrc/gninavis/cnn_visualization.cpp

```diff
@@ -546,7 +546,7 @@ std::string cnn_visualization::get_xyz(const std::string &line)
 {
     std::string x = line.substr(31,7);
     std::string y = line.substr(39,7);
```

Comment:
These aren't right either, I don't think. Should be
x = line.substr(30,8)
y = line.substr(38,8)
z = line.substr(46,9)
trim left and right